web/styles/popovers.css

```diff
@@ -833,14 +833,47 @@ ul {
         margin: 0;
     }
 
-    /* Override the default border-radius to properly align
-       the button corners with `stream_header_colorblock`. */
-    .dropdown-toggle {
-        border-radius: 1px 4px 4px 1px !important;
+    .move-topic-dropdown {
+        display: flex;
+        justify-content: flex-start;
+        margin-bottom: 10px;
+
+        .dropdown-widget-button {
+            /* Override the default border-radius to properly align
+            the button corners with `stream_header_colorblock`. */
+            border-radius: 1px 4px 4px 1px !important;
+            outline: none;
+            line-height: 24px;
+            width: auto;
+            max-width: 206px;
```

Comment:
Can you add a comment explaining where this `max-width` value comes from? I'm also curious about the `line-height`, which I see get used a lot to set space around non-text elements, where `height` or `padding` might be a more consistent option.

Reply:
Added a separate commit for it. Thanks for pointing out!

Reply:
I am using `line-height` match the height of the dropdown widget with that of topic name input box.